cli/Cargo.toml

```diff
@@ -51,6 +51,7 @@ features = ["admin-service", "registry"]
 
 [dependencies.scabbard]
 path = "../services/scabbard/libscabbard"
+default-features = false
```

Comment:
Is this necessary (or should we adjust libscabbard's default feature list instead)?

Reply:
This does ensure that we don't pull in anything that might be added to default in the future.

Reply:
Not sure what we would be optimizing for; if size, then I think the compiler is good enough it will exclude code that isn't used anyway. From that perspective, the only reason for ever turning off default features is if default features pulled in a lot of additional crates to compile.

Reply:
Dropped this commit